src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
```

Comment:
`catalog.scss`
`on 5th line`: don't hardcode your catalog `width` it must depends on your count of card in the row and their width
`on 6th line`: you can't set `margin` to bem-block